YOLO/Info.plist

```diff
@@ -21,7 +21,7 @@
 	<key>CFBundleShortVersionString</key>
 	<string>$(MARKETING_VERSION)</string>
 	<key>CFBundleVersion</key>
-	<string>24</string>
+	<string>354</string>
```

Comment:
Why are we changing the build number?

Reply:
I reverted build number.

Reply:
Is there any problem if you publish an app with the same build number as the previous one?

Reply:
If we have published our app on App Store Connect and we want to publish code changes as an update, we need to change the version or build.

Reply:
I think in this case adding a +1 to this number is the best solution since once this PR is merged, we publish the update.